svgdigitizer/electrochemistry/cv.py

```diff
@@ -599,8 +602,87 @@ def plot(self):
         plt.ylabel(self.axis_properties['y']['dimension'] + ' [' + str(self.axis_properties['y']['unit']) + ']')
 
     @property
-    def metadata(self, comment=''):
-        # TODO: Metadata is not a property (see issue #85)
+    @cache
+    def comment(self):
+        r"""
+        Return a comment describing the plot the plot.
+
+        The comment is read from a ``<text>`` in the SVG file such as ``<text>comment: noisy data</text>``.
+
+        EXAMPLES:
+
+        The first example contains a comment::
+
+            >>> from svgdigitizer.svg import SVG
+            >>> from svgdigitizer.svgplot import SVGPlot
+            >>> from svgdigitizer.electrochemistry.cv import CV
+            >>> from io import StringIO
+            >>> svg = SVG(StringIO(r'''
+            ... <svg>
+            ...   <g>
+            ...     <path d="M 0 200 L 0 100" />
+            ...     <text x="0" y="200">x1: 0 cm</text>
+            ...   </g>
+            ...   <g>
+            ...     <path d="M 100 200 L 100 100" />
+            ...     <text x="100" y="200">x2: 1cm</text>
+            ...   </g>
+            ...   <g>
+            ...     <path d="M -100 100 L 0 100" />
+            ...     <text x="-100" y="100">y1: 0</text>
+            ...   </g>
+            ...   <g>
+            ...     <path d="M -100 0 L 0 0" />
+            ...     <text x="-100" y="0">y2: 1 A</text>
+            ...   </g>
+            ...   <text x="-200" y="330">scan rate: 50 V/s</text>
+            ...   <text x="-400" y="430">comment: noisy data</text>
+            ... </svg>'''))
+            >>> cv = CV(SVGPlot(svg))
+            >>> cv.comment
+            'noisy data'
+
+        The seconmd example does not contain a comment::
```

Comment:
```suggestion
        This example does not contain a comment::
```

There was a typo here. Also, the "first" above confused me because I thought there was more than one in the first section.